projects/ng-components-docs/src/app/examples/components/radio-group/examples/radio-groups/radio-groups.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component } from "@angular/core";
+import { RadioGroupItem } from "lib/src/radio-group/radio-group.component";
+import { FormControl } from "@angular/forms";
+
+@Component({
+    selector: "app-radio-groups",
+    templateUrl: "./radio-groups.component.html",
+})
+export class RadioGroupsComponent {
+    list: RadioGroupItem[] = [
```

Comment:
Why not ?
```
    list: Array<RadioGroupItem> = [
```

Reply:
I'm not a fan of that. `RadioGroupItem[]` is easier to read and takes less space. Imagine you have `Array<Array<Item>>` ... this would be `Item[][]` instead